### PR TITLE
Add Technica orb to navbar

### PR DESCRIPTION
**Pull request by julesparra**

### CODE REVIEW

File: scss/main.scss [hunk 1]
```diff
@@ -151,12 +167,14 @@ $animation-length: 80s;
...
   animation-delay: -$animation-length*$percentange;
+  // freeze to test cloud position
+  //animation-play-state: paused;
 }
 
 // start with cloud 1 at end of screen and stagger from there
 #cloud1 {
   width: 700px;
-  margin-top: -30px;
+  margin-top: 30px;
```
Comment (apanpanpan) [inline scss/main.scss:177]: nit: lets change this to 0px to make things less crowded

File: scss/main.scss [hunk 2]
```diff
@@ -86,6 +86,22 @@ $animation-length: 80s;
   }
 }
 
+.logo {
+  background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
```
Comment (apanpanpan) [inline scss/main.scss:90]: hmm do you think it looks out of place before the white background shows up? we could potentially only add this background attribute in after the white background shows up by styling it in the "scrolled" class. idk what are your thoughts?

File: scss/main.scss [hunk 3]
```diff
@@ -86,6 +86,22 @@ $animation-length: 80s;
...
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+}
+
+.navbar-brand {
```
Comment (apanpanpan) [inline scss/main.scss:96]: the brand logo messes up the centering of the links - lets add `position: absolute` to this class

File: scss/main.scss (hunk 2 above)
Comment (julesparra) [reply scss/main.scss:90]: yep def does, i meant to leave a comment abt it lol -- colors are too harsh in contrast to the pastel bg

File: css/main.css [hunk 4]
```diff
@@ -27,8 +27,8 @@
   background-color: none;
   transition: background-color 200ms linear;
   display: flex;
-  padding-top: 0.01rem;
-  padding-bottom: 0.01rem; }
+  padding-top: 1%;
+  padding-bottom: 1%; }
```
Comment (julesparra) [inline css/main.css:31]: i just realized this left a white bar -- thoughts on how to make the orb feel less squished from top & bottom without messing this up?

File: scss/main.scss [hunk 5]
```diff
@@ -90,10 +90,28 @@ $animation-length: 80s;
...
   box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.05);
   transition: background-color 200ms linear;
+
   .nav-link {
     color: $lavender !important;
     transition: color 200ms linear;
   }
+    
+  .logo {
+    background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
```
Comment (julesparra) [inline scss/main.scss:100]: i feel like the orb colors are still harsh in contrast to pastel theme... i know it's official branding tho so not sure if can change anything

Comment (apanpanpan) [reply scss/main.scss:100]: lol yeah idont think theres anything we can do abt that :/

File: scss/main.scss [hunk 6]
```diff
@@ -90,10 +90,28 @@ $animation-length: 80s;
...
+    width: 50px;
+    height: 50px;
+    border-radius: 50px;
+  }
+
+  .navbar-brand {
+    margin-left: 10px;
+    margin-right: 10px;
+    position: absolute;
```
Comment (apanpanpan) [inline scss/main.scss:109]: lets add the position: absolute outside of the scrolled class so the navbar links don't randomly jump after the white background appears

File: scss/main.scss [hunk 7]
```diff
@@ -67,8 +67,8 @@ $animation-length: 80s;
   background-color: none;
   transition: background-color 200ms linear;
   display: flex;
-  padding-top: 0.01rem;
-  padding-bottom: 0.01rem;
+  padding-top: 1%;
+  padding-bottom: 1%;
```
Comment (apanpanpan) [inline scss/main.scss:71]: i think fixed padding might be the move here or else the navbar can get too fat/skinny, maybe like 5px?

File: scss/main.scss [hunk 8]
```diff
@@ -90,10 +90,28 @@ $animation-length: 80s;
...
   .nav-link {
     color: $lavender !important;
     transition: color 200ms linear;
   }
+    
+  .logo {
+    background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
+    width: 50px;
+    height: 50px;
+    border-radius: 50px;
```
Comment (apanpanpan) [inline scss/main.scss:103]: lets add a media query here for 767px to decrease the size of the orb to match with the decreased font size of the navbar links

File: css/main.css (hunk 4 above)
Comment (apanpanpan) [reply css/main.css:31]: oh lol dont worry abt that whitespace - thats bc i have some hardcoded things for the "About" section which i can always fix

**Pull request merged**